test/test_transforms.py

```diff
@@ -150,6 +150,17 @@ def test_randomresized_params(self):
             assert (min(aspect_ratio_range) - epsilon <= aspect_ratio_obtained <= max(aspect_ratio_range) + epsilon or
                     aspect_ratio_obtained == 1.0)
 
+    def test_randomperspective(self):
+        for i in range(10):
+            height = random.randint(24, 32) * 2
```

Comment:
maybe a more robust test would be to perform a perspective transformation on a few pre-defined points, then invert it and compare that they give similar results (up to an eps)? Similar to what is done in https://stackoverflow.com/questions/14177744/how-does-perspective-transformation-work-in-pil

Reply:
Agreed ! will change it.